README.md

```diff
@@ -1 +1,76 @@
+```
+
+With yarn
+
+```bash
+yarn add @fluencelabs/fluence-network-environment
+```
+
+## Usage
+
+Pick a node to connect to the Fluence network.
+
+```typescript
+import { dev } from '@fluencelabs/fluence-network-environment';
+
+export const relayNode = dev[0];
+```
+
+Which can be used to initialize the Fluence client (see [Fluence JS SDK](https://github.com/fluencelabs/fluence-js).)
+
+```typescript
+import { createClient } from '@fluencelabs/fluence';
+
+const client = await createClient(relayNode);
+```
+
+## Known networks
+
+### dev
+
+**Unstable** Several nodes running on the same physical host. Used a lot to test things out. 
+
+### stage
+
+Environment primarily used for staging deployments of various Fluence services.
+
+### FaaS Net
+
+The main Fluence network. It consists of a ten nodes each running on a separate physical host.
+
+
+## About Fluence
+
+Fluence is an open application platform where apps can build on each other, share data and users
+
+|         Layer         |                             Tech                             |              Scale               |               State               |                           Based on                           |
+| :-------------------: | :----------------------------------------------------------: | :------------------------------: | :-------------------------------: | :----------------------------------------------------------: |
+|       Execution       |          [FCE](https://github.com/fluencelabs/fce)           |           Single peer            | Disk, network, external processes | Wasm, [IT](https://github.com/fluencelabs/interface-types), [Wasmer\*](https://github.com/fluencelabs/wasmer) |
+|      Composition      |   [Aquamarine](https://github.com/fluencelabs/aquamarine)    |          Involved peers          |      Results and signatures       |                        ⇅, π-calculus                         |
+|       Topology        | [TrustGraph](https://github.com/fluencelabs/fluence/tree/master/trust-graph), [DHT\*](https://github.com/fluencelabs/rust-libp2p) | Distributed with Kademlia\* algo |    Actual state of the network    |       [libp2p](https://github.com/libp2p/rust-libp2p)        |
+| Security & Accounting |                          Blockchain                          |          Whole network           |        Licenses & payments        |                          substrate?                          |
+
+<img alt="aquamarine scheme" align="center" src="doc/stack.png"/>
+
+
+
+
+## Contributing
+
+While the project is a still in the early stage of development, you are welcome to track progress and contribute. At the current moment we don't have detailed instructions on how to join development or which code guidelines to follow. However, you can expect more info to appear soon enough. In the meanwhile, check out the [basic contributing rules](https://github.com/fluencelabs/fluence/blob/trustless_computing/CONTRIBUTING.md).
```

Comment:
The link to contributing is strange, it points to the `trustless_computing` tag. Maybe point to master instead?
https://github.com/fluencelabs/fluence/blob/master/CONTRIBUTING.md

Reply:
Hhhm. It might be better to just remove contributing file since this repository is not really meant for contributions

Reply:
Either way, I think we should the same contribution guidelines for all of the Fluence Project repositories... May be it's better to extract this info into readme.io?

Reply:
Not sure, I think the contribution flow needs more love than CONTRIBUTING.md anyway. 

I think removing is a good choice right now.

Reply:
Btw, I've copy-pasted it from fluence rep